app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -61,6 +66,30 @@ public void goToSignUp() {
         SignUpActivity.start(requireContext());
     }
 
+    @Override
+    public void showInvalidCredentialsMessage() {
+        CharSequence message = "Invalid email or password";
```

Comment:
This kind of messages must be saved in the string resources and consumed from them